src/web/client/context/userDataMap.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+export interface IUserData {
+    containerId: string;
+    fileName: string;
```

Comment:
Do fileName and filepath need to be array? We will support user support across multiple files right - how are we handling surfacing that information?

Reply:
address when implement active editor user list